### ISSUE TITLE
rustfmt doesn't recognize efiapi calling convention

### ISSUE THREAD

**Issue by upsuper** [rust-lang/rustfmt#4035]
There was a similar report #1375, maybe the solution is similar to just upgrade syntax dependencies which is undergoing in #4022. This would probably be fixed by that?

**Comment by upsuper** [comment]
The `efiapi` was added in https://github.com/rust-lang/rust/pull/65809

**Comment by calebcartwright** [comment]
Believe this is a duplicate of #3903, but yes, this will be resolved once #4022 is merged (and then released)

**Comment by upsuper** [comment]
Yeah it's probably a duplicate of that.
